file-check/action.yaml

```diff
@@ -0,0 +1,54 @@
+name: file-check
+description: ""
+
+inputs:
+  files:
+    description: ""
+    required: true
+  condition:
+    description: ""
+    required: true
+outputs:
+  exists:
+    description: ""
+    value: ${{ steps.file-check.outputs.exists }}
+
+runs:
+  using: composite
+  steps:
+    - name: File check
+      id: file-check
+      run: |
+        set -e
+
+        # Split files into an array
+        input_files="${{ inputs.files }}"
```

Comment:
Why do you need this local variable?

Can't you write like this?

```bash
input_files=("${{ inputs.files }}")
```

Reply:
It doesn't work correctly. So I used local variable.

Reply:
Okay, I understand.